pages/books/BookList.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { useTheme } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+
+export default function BookList() {
```

Comment:
now that we have seed data, we should hook up some data from the backend here

Reply:
okay will check it out

Reply:
see comment here for this file - https://github.com/sprathap1096/CS554_Final_Project_Emanon/pull/5#discussion_r867399376